cmd/operator/main.go

```diff
@@ -376,12 +376,22 @@ func Main() int {
 		}
 	}
 
-	validationTriggeredCounter := prometheus.NewCounter(prometheus.CounterOpts{
+	promValidationTriggeredCounter := prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_operator_rule_validation_triggered_total",
+		Help: "Number of times a prometheusRule object triggered validation",
+	})
+
+	promValidationErrorsCounter := prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "prometheus_operator_rule_validation_errors_total",
+		Help: "Number of errors that occurred while validating a prometheusRules object",
+	})
+
+	promRuleValidationTriggeredCounter := prometheus.NewCounter(prometheus.CounterOpts{
 		Name: "prometheus_operator_rule_validation_triggered_total",
 		Help: "DEPRECATED, removed in v0.57.0: Number of times a prometheusRule object triggered validation",
```

Comment:
Hmmm, looks like we deprecated the metrics itself? From a [codesearch](https://github.com/search?q=repo%3Aprometheus-operator%2Fprometheus-operator+validationErrorsCounter&type=code), looks like we're not increasing anywhere in the code. Is that correct @simonpasquier, @slashpai? What are we using instead?

Reply:
the Kubernetes API already exposes metrics about webhooks (e.g. `apiserver_admission_webhook_admission`_duration) so there's no need for the operator to repeat them. I'd be fine dropping the deprecated metrics in the operator (in a separate PR).

Reply:
@rexagod can you open another PR to remove the deprecated metrics as a first step? It'd make this PR a bit smaller.

Reply:
> From a [codesearch](https://github.com/search?q=repo%3Aprometheus-operator%2Fprometheus-operator+validationErrorsCounter&type=code), looks like we're not increasing anywhere in the code.

[IIUC, it seems they are.](https://github.com/prometheus-operator/prometheus-operator/pull/6024/files#diff-b51086a9eecc900f44fb39e672c73f66396d84c48d15f5d86c8106cea04a4920L270) I've opened up the PR just in case we still want to work towards deprecating these metrics (or not).

Reply:
Deprecated metrics have been dropped in #6024.